sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -44,8 +44,10 @@ def test_missing_from_config_file_throws(self, _, missing_attributes, token):
 
         api_config_kwargs = {key: value for key, value in source_config_details.items() if
                              value is not None and "proxy" not in key}
+
         api_config_kwargs["proxy_config"] = proxy_config
         api_configuration = ApiConfiguration(**api_config_kwargs)
+        api_configuration.access_token=None
 
```

Comment:
The pat access token is now loaded into the **api_configuration**, so we need to excluded the pat token from the test.